pkgs/development/python-modules/stem/default.nix

```
@@ -26,6 +26,5 @@ buildPythonPackage rec {
     homepage = https://stem.torproject.org/;
```

```suggestion
    homepage = "https://stem.torproject.org/";
```

Oops, didn't actually make my suggestion... lol

---

this change was already introduced in the master branch,
plus there was also a version bump.

---

Heh, I didn't notice the base branch (thought this was against master). Sorry for the noise.

---

does this mean that this conversation can be resolved?